prediction_market_agent/agents/replicate_to_omen_agent/deploy.py

```diff
@@ -56,22 +66,34 @@ def run(self, market_type: MarketType = MarketType.MANIFOLD) -> None:
         logger.info("Redeeming funds from previously unfunded markets.")
         redeem_from_all_user_positions(keys)
 
-        for replicate_from_market_type in [MarketType.MANIFOLD, MarketType.POLYMARKET]:
-            for close_time_days in settings.CLOSE_TIME_UP_TO_N_DAYS:
-                close_time_before = utcnow() + timedelta(days=close_time_days)
+        for replicate_config in settings.REPLICATE:
+            if now.timetuple().tm_yday % replicate_config.every_n_days:
```

Comment:
Shouldn't this be 
```suggestion
            if now.timetuple().tm_yday % replicate_config.every_n_days > 0:
```
?

e.g. if `replicate_config.every_n_days == 1` then this will always skip!

Reply:
Oops sorry, had it the wrong way round in my head!